### PR TITLE
switch to only supporting node versions greater than 12, update deps

### PR DESCRIPTION
**Pull request by jdelbick**
## Description
I couldn't update a lot of the deps because the newest versions only support ESM:
- ex: https://github.com/TooTallNate/node-data-uri-to-buffer/releases/tag/4.0.0

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
     "type": "git",
     "url": "adobe/asset-compute-sdk"
   },
+  "engines": {
+    "node": ">=12.0.0"
+  },
```
Comment (jdelbick) [inline package.json:15]: this ensures it can only be installed with node >=12

Comment (tmathern) [reply package.json:15]: And we need this because there is a breakage on node 10? Or just because we don't support node 10 anymore?

**Comment by jdelbick** [comment]
would this technically be a breaking release? wdyt? @tmathern @pheenomenon

**Comment by pheenomenon** [comment]
As we would cause breakage for anyone on node 10,  I would go with a breaking change.

File: package.json (hunk 1 above)
Comment (pheenomenon) [reply package.json:15]: I think because there is a breakage on node 10? with the dips updates.. , is that so @jdelbick ?

**Review by pheenomenon — APPROVED** [review]

**Pull request closed without merging**